tests/testsuite/fix.rs

```diff
@@ -1942,3 +1942,107 @@ fn fix_only_once_for_duplicates() {
             1 from newly-applied unsafe blocks"
     );
 }
+
+#[cargo_test(nightly, reason = "edition2024")]
+fn migrate_project_to_package() {
+    let p = project()
+        .file(
+            "Cargo.toml",
+            r#"
+cargo-features = ["edition2024"]
+
+# Before project
+[ project ] # After project header
```

Comment:
why do we need extra spaces in-between?

Reply:
I put in as much non-standard formatting as I could think of to highlight that we are format-preserving